libs/net/dal/Configuration/ContentConfiguration.cs

```diff
@@ -37,7 +37,8 @@ public override void Configure(EntityTypeBuilder<Content> builder)
         builder.HasMany(m => m.Tags).WithMany(m => m.Contents).UsingEntity<ContentTag>();
         builder.HasMany(m => m.TonePools).WithMany(m => m.Contents).UsingEntity<ContentTonePool>();
 
-        builder.HasIndex(m => new { m.Status, m.PublishedOn, m.OtherSource, m.Uid, m.Page });
+        builder.HasIndex(m => new { m.PublishedOn, m.CreatedOn });
```

Comment:
Don't know if this is better performance, but this is our most common index so I separated it.